src/operators.jl

```diff
@@ -137,6 +137,23 @@ function union{T<:SimpleGraph}(g::T, h::T)
     return r
 end
 
+
+"""Merges graphs `g` and `h`. The resulting graph has `nv(g)+nv(h)``
+    and ne(g)+ne(h) edges. Vertex indexes in `h` are shifted by `nv(g)``.
+"""
+function join{T<:SimpleGraph}(g::T, h::T)
```

Comment:
Ops, sorry, I didn't notice. Also, `join` is unfortunate, since in graph literature it means something else http://mathworld.wolfram.com/GraphJoin.html. I'll revert to `blkdiag`, unless we choose a different name for it, since it sounds a little awkward to me (but it could be familiar to matlab's users)